fdbclient/ServerKnobs.cpp

```diff
@@ -620,6 +620,9 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( BLOB_GRANULE_LOCATION_MAX_QUEUE_SIZE,                  1e5 ); if ( randomize && BUGGIFY ) BLOB_GRANULE_LOCATION_MAX_QUEUE_SIZE = 100;
 	init( COMMIT_PROXY_LIVENESS_TIMEOUT,                        20.0 );
 
+	// FIXME Remove this when client latency issue is resolved
+	init( WRITE_CLIENT_LATENCY_TRACEEVENT,                       true );
```

Comment:
do we want the default to be false?

Reply:
Let's set it to be true at this stage, Personally I would guess the load would not impact the system.